Clocks/FMI3.xml

```diff
@@ -27,10 +27,13 @@
   <Output valueReference="2004" dependencies="1001 1002 1003"/>
   <Output valueReference="2005" dependencies="1002 2006"/>
   <Output valueReference="2007" dependencies="1003"/>
-  <Clock valueReference="1001" clockedVariables="1005 2001 2004"/>
-  <Clock valueReference="1002" clockedVariables="1005 2002 2005"/>
-  <Clock valueReference="1003" clockedVariables="1005 2004 2007"/>
-  <Clock valueReference="1005"/>
+  <InitialUnknown valueReference="2001"/>
+  <InitialUnknown valueReference="2002"/>
+  <InitialUnknown valueReference="2003"/>
+  <InitialUnknown valueReference="1005"/>
```

Comment:
`  <InitialUnknown valueReference="1005"/>` should not be there.
  Clocks should not be in initalUnknonws.